pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -46,10 +49,16 @@ func NewPgxIngestor(conn pgxconn.PgxConn, cache cache.MetricCache, sCache cache.
 	if err != nil {
 		return nil, err
 	}
+
+	batcherConfg := trace.BatcherConfig{
+		MaxBatchSize: cfg.TracingMaxBatchSize,
+		BatchTimeout: cfg.TracingBatchTimeout,
+		Writers:      cfg.NumCopiers,
```

Comment:
Right, as you've noticed as it is now traces and metrics share the same pool of writers. Since not many users use ingest metrics/traces at the same time I haven't been focusing much on this. I'll create an issue to keep a track of this.